pkg/collector/collector.go

```diff
@@ -1,88 +1,54 @@
 package collector
 
 import (
+	"github.com/archway-network/relayer_exporter/pkg/ibc"
 	log "github.com/archway-network/relayer_exporter/pkg/logger"
-	"github.com/archway-network/relayer_exporter/pkg/relayer"
+	gorelayer "github.com/cosmos/relayer/v2/relayer"
 	"github.com/prometheus/client_golang/prometheus"
 	"go.uber.org/zap"
 )
 
 const (
-	clientExpiryMetricName    = "cosmos_relayer_client_expiry"
+	clientExpiryMetricName    = "cosmos_ibc_client_expiry"
 	configuredChainMetricName = "cosmos_relayer_configured_chain"
 	relayerUpMetricName       = "cosmos_relayer_up"
 )
 
-var (
-	up = prometheus.NewDesc(
-		relayerUpMetricName,
-		"Was talking to relayer successful.",
-		nil, nil,
-	)
-
-	clientExpiry = prometheus.NewDesc(
-		clientExpiryMetricName,
-		"Returns light client expiry in unixtime.",
-		[]string{"chain_id", "path"}, nil,
-	)
-
-	configuredChain = prometheus.NewDesc(
-		configuredChainMetricName,
-		"Returns configured chain.",
-		[]string{"chain_id"}, nil,
-	)
+var clientExpiry = prometheus.NewDesc(
+	clientExpiryMetricName,
+	"Returns light client expiry in unixtime.",
+	[]string{"chain_id", "client_id", "path"}, nil,
 )
```

Comment:
perhaps using 'host_chain_id instead' of 'chain_id' and 'target_chain_id instead' of 'path' would be more explicit and descriptive here.

Reply:
What about labels like `chain_id`, `client_id` and `target_chain_id`?  With `host_chain_id` I'm not sure it is obvious that `client_id` is for client on `host_chain_id`